lib/project_types/script/messages/messages.rb

```diff
@@ -50,6 +50,25 @@ module Messages
 
           script_not_found_cause: "Couldn't find script %s for extension point %s",
 
+          service_failure_cause: "Internal service error. " \
+                                 "Ensure the 'shopify/scripts-toolchain-as' package is up to date.",
+
+          metadata_validation_cause: "Invalid script extension metadata. " \
+                                     "Ensure the 'shopify/scripts-toolchain-as' package is up to date and " \
+                                     "'package.json' contains a 'scripts/build' entry with a " \
+                                     "'--metadata build/metadata.json' argument",
```

Comment:
This error probably probably wouldn't be due to a missing metadata flag, that likely would result in the metadata file not being found.